naas/models/project_subscriber_profile.py

```diff
@@ -0,0 +1,114 @@
+import naas
+from naas.configuration import Configuration
+from naas.errors import InvalidRequestError
+from naas.models.error import Error
+from naas.models.project import Project
+from naas.models.projects import Projects
+from naas.models.project_subscriber import ProjectSubscriber
+from naas.models.project_subscribers import ProjectSubscribers
+
+
+class ProjectSubscriberProfile(object):
+    """
+
+    Project Subscriber Profile
+    ===============
+
+    This returns an instance of the Project Subscriber Profile domain model
+    """
+
+    def __init__(self, attributes={}):
+        self.attributes = attributes
+        self.project_id = self.attributes.get('project_id')
+        self.project_subscriber_id = self.attributes.get('project_subscriber_id')
+        for key, value in self.attributes.items():
+            setattr(self, key, value)
+            getattr(self, key)
```

Comment:
This is the wrong implementation. It's not _deleting_ the `project_id` and `project_subscriber_id` and then using the attributes. It's setting them all, which then causes errors later in the code when trying to use the `project_id()`